### ISSUE TITLE
Object.prototype.toString.call does not behave as expected

### ISSUE THREAD

**Issue by courtnek** [ariya/phantomjs#11722]
Filing an issue to track the following behavior (tested with Phantom 1.6.0):

http://stackoverflow.com/questions/14218670/why-are-null-and-undefined-of-the-type-domwindow

phantomjs> Object.prototype.toString.call(undefined);
"[object DOMWindow]"

**Comment by kibin** [comment]
Any updates on this? I have this issue on 1.9.8

**Comment by LeMoussel** [comment]
On Windows 7, with PhantomJS 2.0.0 I haven't this issue
```
phantomjs> Object.prototype.toString.call(undefined);
"[object Undefined]"
phantomjs>
```

**Comment by Vitallium** [comment]
REPL is known to be broken in 1.9.X version.

**Comment by kibin** [comment]
Hm, I’m using it with karma, which installs 1.9.X by default. Will try to switch to 2.0, thanks.

**Comment by Vitallium** [comment]
Fixed in 2.0.
